src/gui/optionsdialog.ui

```diff
@@ -837,6 +837,9 @@
             </item>
             <item>
              <widget class="QCheckBox" name="checkRecursiveDownload">
+              <property name="toolTip">
+               <string>When a torrent is downloaded, ask to download any .torrent files within the torrent</string>
```

Comment:
It confuses me. I think "download any .torrent files" is not what you really want to do. At least I understand "download .torrent files" literally. Shouldn't it be "add torrents from any .torrent files" instead?

Reply:
@glassez, @adem4ik:  Honestly, I was never quite satisfied with the tooltip text, either.  How about… 

> When a torrent is downloaded, **open add torrent dialog for** any .torrent files within the torrent 

…or… 

> When a torrent is downloaded, **open add torrent dialog for** any .torrent files **found inside**

Reply:
@LordNyriox I like the second one, cause it has less repeating of "torrent" word" :D
> When a torrent is downloaded, open add torrent dialog for any .torrent files found inside

Reply:
>When a torrent is downloaded, open add torrent dialog for any .torrent files found inside

But it doesn't really open that dialog, it opens another dialog:
![pic](https://user-images.githubusercontent.com/9395168/70294108-6f617e80-181d-11ea-8797-71373750641c.png)

Reply:
> it doesn't really open that dialog, it opens another dialog 

Ah, OK.  I was not really certain of that part.  Thanks for the correction, @Chocobo1! 

> When a torrent is downloaded, **offer to download** any .torrent files **found inside it**

Reply:
@adem4ik 
You can update this PR like before and ping me for the final squash.

Reply:
@Chocobo1 PR is updated.